image_setup/02_devel_image/Dockerfile

```diff
@@ -5,13 +5,27 @@ FROM ${BASE_IMAGE_NAME}
 # To avoid some errors when fetching apt-get updates. # See: https://stackoverflow.com/a/35976127 
 ENV DEBIAN_FRONTEND=noninteractive
 
+# generic packages to install non-workspace dependencies ( those are added later )
 RUN apt-get update && apt-get install -y \
     bash-completion
 
+# setup locale to avoid ASCII conflicts
+RUN locale-gen en_US.UTF-8
+ENV LANG="en_US.UTF-8"
+
+# make sure SHELL variable is set. Crucial for base/scripts! => moved to setup_workspace.bash
+## REMOVE THIS ONCE rock-core PR 173 is merged
+ENV SHELL="/bin/bash"
+
 # add workspace init to container initialization
 ADD setup_workspace.bash /opt/setup_workspace.bash
 RUN chmod 755 /opt/setup_workspace.bash
 
+# add the dependency list to the image
+ADD workspace_os_dependencies.txt /opt/workspace_os_dependencies.txt
```

Comment:
Doesn't this conflict with mounting this file through exec.bash? I get that this is required for the next line, but shouldn't we work on the mounted file version and cat the file on the host system instead? Or do we need the file inside the image to cat it?

Would this work?
```Dockerfile
RUN cat workspace_os_dependencies.txt | xargs sudo apt install -y
```

Reply:
There is no conflict in that, when the file is mounted the mounted version is used, RUN is executed in a container, so workspace_os_dependencies.txt must be available. If we want to be sure, we could:

    RUN cat workspace_os_dependencies.txt | xargs sudo apt install -y && rm -f workspace_os_dependencies.txt

Reply:
renamed the ADDed file to installed_workspace_os_dependencies.txt, which 
 now allows to see what is actually installed and to compare with a newly generated file to see if a image update is needed